svelte.config.js

```diff
@@ -30,7 +30,9 @@ const config = {
           filepath,
         );
       },
-      files: (filepath) => /^(?!.*(?:test)).*\.ts$/.test(filepath),
+      // files: (filepath) => true,
```

Comment:
maybe a little remnant of testing locally? 🚮

Reply:
Indeed, I'll take the trash out.